docs/usage.rst

```diff
@@ -204,6 +204,31 @@ Now we're going to get the base URL and some image links::
      'image4_thumb.jpg',
      'image5_thumb.jpg']
 
+JMESPath Selector
+-----------------
+Whenever the selected elements possess JSON format, it is possible to use  ``.jmespath()`` method from
```

Comment:
Instead of “selected elements” (“element” here can be easily confused with “HTML/XML element”), I think we should talk of “selector text” (the string the Selector selects from).

“possess” suggests having JSON (i.e. possibly only partially), “is” is probably a better choice.

```suggestion
Whenever the selector text is JSON, it is possible to use the ``.jmespath()`` method from
```